Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -223,7 +235,8 @@
         "UnabletofindthelocationofMSDeployfromregistryonmachineError": "Unable to find the location of MS Deploy from registry on machine (Error : %s)",
         "Nopackagefoundwithspecifiedpattern": "No package found with specified pattern",
         "MorethanonepackagematchedwithspecifiedpatternPleaserestrainthesearchpatern": "More than one package matched with specified pattern. Please restrain the search patern.",
-        "Trytodeploywebappagainwithappofflineoptionselected": "Try to deploy web app again with app_offline option selected.",
+        "Trytodeploywebappagainwithappofflineoptionselected": "Try to deploy app service again with app_offline option selected.",
+		"Trytodeploywebappagainwithrenamefileoptionselected": "Try to deploy app service again with Rename locked files option selected.",
```

Comment:
Help says Rename locked files, but the label for the option is Rename files. Can you make it consistent.